### ISSUE TITLE
JAWS Freezes When Closing Sds Tree Table using the SdsDialogService

### ISSUE THREAD

**Issue by Sunset05** [GSA/sam-design-system#1277]
**Components Involved**
Sds Tree Table
https://gsa.github.io/sam-design-system/?path=/docs/components-tree-table--introduction
Sds Dialog Service
https://gsa.github.io/sam-design-system/?path=/docs/components-dialog--introduction

**Expected Behavior**
when closing the dialog service and tree table JAWS is able to focus back on the main content of the page that opened the modal and tree table

**Actual Behavior**
Currently when closing the Dialog and tree table, JAWS freezes and is unable to recognize where on the page it should focus

**Your Environment**
Angular Version: 14.3.0
@gsa-sam/components: 14.0.1
@gsa-sam/layout: 14.0.10
@gsa-sam/layouts: 14.0.3
@gsa-sam/ngx-uswds: 14.0.2
@gsa-sam/ngx-uswds-icons: 14.0.0-beta.2
@gsa-sam/sam-formly: 14.0.1
@gsa-sam/sam-material-extensions: 14.0.1
@gsa-sam/sam-styles: 3.0.7
@gsa-sam/shared: 14.0.1

**Ticket Number**
Please provide your JIRA ticket number that is dependent on this fix. If there is none, please create
one to validate the issue once the fix has been released. DO NOT provide the url, please ONLY
provide the ticket number.
IAEMOD-18549

**Additional context**
To reproduce - using JAWS close the NAICS tree table modal under Reps and Certs provision FAR 52.525-3 on the /entity-360/entity1/UEI/repsAndCerts? end point
 
![image](https://github.com/GSA/sam-design-system/assets/77186128/a5f9dfa5-623d-4b07-8883-ddd3984b57a0)
![image](https://github.com/GSA/sam-design-system/assets/77186128/c0ab08a2-c4b2-4371-b7d9-e1550ddeea9a)
![image](https://github.com/GSA/sam-design-system/assets/77186128/ddc26ca4-5ae6-4b5e-b68f-44a45f3b6c25)
